modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -3,127 +3,59 @@
+
 .Procedure
 
-. Customize the main menu items using the following steps:
+. To configure a Dynamic plugin menu item, add the following content to your `{my-app-config-file}` file:
 +
---
-.. Open the `app-config-rhdh.yaml` file.
-.. To customize the order and parent-child relationships for the main menu items, use the `dynamicPlugins.frontend.default.main-menu-items.menuItems` field.
-.. For dynamic plugin menu items, use the `dynamicPlugins.frontend.<package_name>.menuItems` field.
-
-.Example `app-config-rhdh.yaml` file
 [source,yaml]
 ----
 dynamicPlugins:
   frontend:
-    <package_name>: # same as `scalprum.name` key in plugin's `package.json`
-      menuItems: # optional, allows you to configure plugin menu items in the main sidebar navigation
-        <menu_item_name>: # unique name in the plugin menu items list <1>
-          icon: home | group | category | extension | school | _<your_icon>_  # <2>
-          title: My Plugin Page # optional, same as `menuItem.text` in `dynamicRoutes` <3>
-          priority: 10 # optional, defines the order of menu items in the sidebar <4>
-          parent: favorites # optional, defines parent-child relationships for nested menu items <5>
-----
-
-You can modify the fields in the previous example to configure the desired order and parent-child relationships of the sidebar menu items.
-
-<1> This attribute represents a unique name in the main sidebar navigation. It can denote either a standalone menu item or a parent menu item. If this attribute represents a plugin menu item, the name of the attribute must match with the corresponding path in `dynamicRoutes`. For example, if `dynamicRoutes` defines `path: /my-plugin`, then `menu_item_name` must be defined as `my-plugin`.
-+
-For more complex, multi-segment paths such as `path: /metrics/users/info`, the `menu_item_name` must use dot notation to represent the full path, for example, `metrics.users.info`. Trailing and leading slashes in paths are ignored. For example, `path: /docs` results in `menu_item_name: docs`, and `path: /metrics/users` results in `menu_item_name: metrics.users`.
-
-<2> This optional attribute specifies the icon for the menu item. You can use default icons or extend the icon set with dynamic plugins. {product-short} also provides additional icons in its internal library, such as: 
-+
-.Home Icon in the internal library
-[source, yaml]
-----
-dynamicPlugins:
-  frontend:
-    <package_name>:
-      menuItems:
-        <menu_item_name>: 
-          icon: home
-----
-+
-Similarly, the internal library includes icons for `group`, `category`, `extension`, and `school`. If the icon is already defined in the `dynamicRoutes` configuration under `menuItem.icon`, it can be removed from the in the `menuItems` configuration. Also, both SVG and HTML image icons are supported. For example:
+    <package_name>: <1>
```

Comment:
I forgot to mention: do not forget to add a comment before each callout. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml

```suggestion
    <package_name>: # <1>
```